.devcontainer/Dockerfile

```diff
@@ -1,10 +1,18 @@
 # [Choice] Python version (use -bookworm or -bullseye variants on local arm64/Apple Silicon): 3, 3.13, 3.12, 3.11, 3.10, 3.9, 3.8, 3-bookworm, 3.13-bookworm, 3.12-bookworm, 3.11-bookworm, 3.10-bookworm, 3.9-bookworm, 3.8-bookworm, 3-bullseye, 3.11-bullseye, 3.10-bullseye, 3.9-bullseye, 3.8-bullseye, 3-buster, 3.11-buster, 3.10-buster, 3.9-buster, 3.8-buster
-ARG VARIANT=3.12-bookworm
-FROM ghcr.io/astral-sh/uv:python${VARIANT}
-ENV UV_LOCKED=1
-ENV UV_PYTHON=/usr/local/bin/python
+ARG VERSION=3.12
+ARG VARIANT=-bookworm
 
+FROM python:${VERSION}${VARIANT}
+
+ENV UV_LOCKED=1 UV_PYTHON=${VERSION} PATH="/root/.local/bin/:$PATH"
+
+# https://docs.astral.sh/uv/guides/integration/docker/#installing-uv
 RUN apt-get update && export DEBIAN_FRONTEND=noninteractive \
-    && apt-get purge -y fish
+    && apt-get purge -y fish \
+    && apt-get install -y --no-install-recommends curl ca-certificates
+
+ADD https://astral.sh/uv/install.sh /uv-installer.sh
+
+RUN sh /uv-installer.sh && rm /uv-installer.sh
```

Comment:
Maybe better to use `curl` and `rm` in a single run? Now `/uv-installer.sh` will be in `ADD` layer.

Reply:
@sobolevn what about copying as shown [here](https://docs.astral.sh/uv/guides/integration/docker/#installing-uv)? It should avoid the need to download curl and also cleanup?

```
FROM python:3.12-slim-bookworm
COPY --from=ghcr.io/astral-sh/uv:latest /uv /uvx /bin/
```

Reply:
Great idea!